setup.cfg

```diff
@@ -18,7 +18,7 @@ package_dir =
 packages = find:
 python_requires = >=3.8
 install_requires =
-    geoengine-openapi-client == 0.0.4
+    geoengine-openapi-client @ git+https://github.com/1lutz/openapi-client.git/#subdirectory=python # TODO update when merged
```

Comment:
remember to change

Reply:
651f97b0f2113c32e9636ff72423f779f31ea32b